src/services/test-result-record-conversion.ts

```diff
@@ -122,13 +129,28 @@ const upsertTestResults = async (testResults: TestResults): Promise<void> => {
                         undefined,
                         createdById,
                         lastUpdatedById,
+                        moment().format("YYYY-MM-DD HH:mm:ss.SSS"),
                     ],
                     testResultConnection
                 );
 
                 continue;
             }
 
+            if (!process.env.DISABLE_DELETE_ON_UPDATE) {
```

Comment:
This really nice to have. Do you think the feature toggle should be a covered by testing?

Reply:
Done. It was tricky to get both working! 😃
Ran through the pipeline:
![image](https://user-images.githubusercontent.com/13391709/224322282-9eaad9f3-6970-44c7-9dc3-0cb3ce382fd6.png)